app/routes/questions/$question.tsx

```diff
@@ -70,17 +66,7 @@ export function Question({
   glossary: Glossary
   selectQuestion: (pageid: string, title: string) => void
 } & JSX.IntrinsicElements['div']) {
-  const {
-    pageid,
-    title: codaTitle,
-    status: codaStatus,
-    text,
-    answerEditLink,
-    questionState,
-    tags,
-  } = questionProps
-  const title =
-    codaStatus && codaStatus !== QuestionStatus.LIVE_ON_SITE ? `WIP - ${codaTitle}` : codaTitle
```

Comment:
a banner can be displayed for non published questions, rather than the `WIP - `

Reply:
they all have the appropriate banner set, so it's mainly a matter of deciding whether that's the way to go. The parser is already updated to do so

Reply:
oh, this most certainly is a bad choice - I pretty much just plopped random colours there to "encourage" someone else to choose decent ones :D